packages/lesswrong/components/posts/PostsUserAndCoauthors.tsx

```diff
@@ -51,8 +52,14 @@ const PostsUserAndCoauthors = ({post, abbreviateIfLong=false, classes, simple=fa
   
   return <div className={abbreviateIfLong ? classes.lengthLimited : classes.lengthUnlimited}>
     {<UsersName user={post.user} simple={simple} />}
-    {post.coauthors.map(coauthor =>
-      <span key={coauthor._id}>, <UsersName user={coauthor} simple={simple}  /></span>)}
+    {post.coauthors.map(coauthor => postCoauthorIsPending(post, coauthor._id)
```

Comment:
Good point! It's probably a bit less efficient because we have to traverse the list twice, but it's always going to be such a short list that it should be worth it for the sake of cleaner code.